tsl/test/expected/gapfill.out

```diff
@@ -371,6 +371,18 @@ GROUP BY 1,id ORDER BY 2,1;
     4 |  2 |  
 (10 rows)
 
+-- test grouping by non-time columns with no rows in resultset
```

Comment:
I'd like to see a non-multigroup test case with an empty resultset as well

Reply:
added